src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,34 @@
+
+from __future__ import absolute_import, division
+
+from celery.utils.log import get_task_logger
+from django.utils import timezone
+
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.cache import Lock, UnableToGetLock
+
+logger = get_task_logger(__name__)
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock_key = 'scheduler:process'
+    try:
+        with Lock(lock_key, nowait=True, timeout=60):
+            queryset = list(ScheduledJob.objects.filter(
+                date_scheduled__lte=timezone.now(),
+            )[:100])
```

Comment:
I explicitly chose to limit it to make sure we dont have a similar issue we have in the resolution cleanup (that its unbounded). If it gets backlogged that's fine. We could obviously be more smart and have it requeue itself, but I dont foresee this being an issue.